docs: PEM-9658: GHCR Support (#10190)
spectrocloud/librarium: docs/docs-content/registries-and-packs/registries/oci-registry/oci-registry.md

Palette supports the configuration of multiple Open Container Initiative (OCI) registries. You can add a private OCI
registry to Palette and use OCI artifacts in your cluster profiles. Palette only supports custom OCI registries that are
hosting the following type of artifacts: Helm charts, Zarf packages, and Packs.

Registries that comply with the OCI specification and support basic authentication are supported, although you can also
add registries that do not require authentication. OCI-compliant registries with basic authentication are labeled as
**Basic**. Palette also supports certain third-party registry providers, such as Amazon Elastic Container Registry
(ECR), Azure Container Registry (ACR), and GitHub Container Registry (GHCR).

## Supported Authentication Methods

> The following table describes the different authentication methods supported based on the type of OCI registry.

Reviewer: ```suggestion
The following table describes the authentication methods supported for each OCI registry type.
```

| **Helm**     | OCI-based Helm Chart registries |  :white_check_mark:   |    :white_check_mark:    | :white_check_mark: |
| **Zarf**     | OCI-based Zarf registries       |          :x:          |    :white_check_mark:    |        :x:         |
| **Packs**    | OCI-based Packs registries      |          :x:          |    :white_check_mark:    | :white_check_mark: |

:::info

Anonymous authentication is not supported for GHCR.

:::

## Verified OCI Registries

The following OCI registries have been verified by Palette. Other OCI registries may work but have not been verified.

- Amazon ECR
- ACR
- GHCR
- Harbor
- JFrog Artifactory

## Project-Level vs. System-Level Registries

OCI registries are typically added at the project level from the Tenant Admin scope. When a registry is added using this
method, it is available to all projects within the tenant.

If you are using self-hosted Palette or Palette VerteX, you can also add an OCI registry at the system level through the
[system console](../../../enterprise-version/system-management/system-management.md#access-the-system-console). All

> tenants, and all projects within those tenants, can use the OCI registry once it is added at the system level. To learn

Reviewer: ```suggestion
tenants and all their projects can use the OCI registry once it is added at the system level. To learn
```

Author: Using the other suggestion provided, as I feel it is more clear.

> tenants, and all projects within those tenants, can use the OCI registry once it is added at the system level. To learn

Reviewer: ```suggestion
tenants and all projects within those tenants can use the OCI registry once it is added at the system level. To learn
```
